src/libraries/System.Text.Json/tests/Common/CollectionTests/CollectionTests.Specialized.Read.cs

```diff
@@ -0,0 +1,52 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Specialized;
```

Comment:
I assume these tests were just moved...

Reply:
Yup, simply moved to share them with the source generator tests.